docs/en/userguide/faq.md

```diff
@@ -107,7 +107,7 @@ You can recreate the dataset, and the new dataset will display these fields norm
 
 ## 6. Why do I run the example [Nonroot access](../samples/nonroot_access.md), and I  encounter mkdir permission denied error
 
-**Answer**: In nonroot scenirio,Firstly,you have to check that you pass the right useinfo to runtime,Secondly,you should check the alluxio master pod status,and use journalctl to see the kubelet logs in the node of alluxio master pod.the mkdir error was cause when mount the hostpath to container,So we have to check the root have right permission to exec the directory.We can use stat to check if root can exec the directory.For example in the below root have permission to operator /dir
+**Answer**: In nonroot scenario, Firstly, you have to check that you pass the right useinfo to runtime, Secondly, you should check the alluxio master pod status, and use journalctl to see the kubelet logs in the node of alluxio master pod. The mkdir error was caused when mounting the hostpath to the container, So we have to check the root has the right permission to exec the directory. We can use stat to check if root can exec the directory. For example in the below root have permission to operator /dir
```

Comment:
In non-root scenario, you have to check if you pass the right user info(?) to runtime first. Secondly, you should check the alluxio master pod status, and use journalctl to see the kubelet logs in the node of alluxio master pod.  The mkdir error was caused when mounting the hostpath to the container and therefor we have to check the root has the right permission to exec the directory.

Reply:
Fixed, thanks.